source/services/encryption/EncryptionHelper.ts

```diff
@@ -13,19 +13,24 @@ export function EncryptionException(message: string) {
 export function getPublicKeyInForm(
   personalNumber: string,
   forms: AnsweredForm
-) {
+): number {
   return forms.encryption.publicKeys[personalNumber];
 }
 
-function getSymmetricKeyStorageKeyword(forms: AnsweredForm) {
+function getSymmetricKeyStorageKeyword(forms: AnsweredForm): string {
   return forms.encryption.symmetricKeyName;
 }
 
-function getPrivateKeyStorageKeyword(user: UserInterface, forms: AnsweredForm) {
+function getPrivateKeyStorageKeyword(
+  user: UserInterface,
+  forms: AnsweredForm
+): string {
   return `aesPrivateKey${user.personalNumber}${forms.encryption.symmetricKeyName}`;
 }
 
-export async function getStoredSymmetricKey(forms: AnsweredForm) {
+export async function getStoredSymmetricKey(
+  forms: AnsweredForm
+): Promise<number> {
```

Comment:
Sure I'm stupid, but is it correct that this func will return a promise?
Take it that Number is a promise?

Reply:
Every `async` function is automatically "converted" and acts as a Promise, so the correct return type for any `async` function is always `Promise<...>` (in case of no return value, it is still `Promise<void>`). Linting also enforces this by complaining otherwise 🙂.